ui/src/main/webapp/app/saved-results/saved-results-component.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+angular.module('app').component('savedResults', savedResults())
+
+function savedResults() {
+    return {
+        templateUrl: 'app/saved-results/saved-results.html',
+        controller: SavedResultsController,
+        controllerAs: 'ctrl'
+    }   
+}
+
+function SavedResultsController(loading, query, graph, error, navigation, events, $cookies, config) {
+    var saveResultsConfig = {
+        enabled: false
+    };
+
+    var vm = this;
+    vm.savedResults = [];
+
+    /**
+     * Loads the saved results
+     */
+    vm.$onInit = function() {
+        events.subscribe('resultsSaved', function() {
+            vm.savedResults = loadSavedResults();
+        });
+
+        config.get().then(function(conf) {
+            if (conf.savedResults) {
+                saveResultsConfig = conf.savedResults;
+                if(saveResultsConfig.enabled) {
+                    vm.savedResults = loadSavedResults();
+                }
+            }
+        });
+    }
+
+    vm.updateSavedResults = function() {
+        updateSavedResults(vm.savedResults);
+        vm.savedResults = loadSavedResults();
+    }
+
+    vm.deleteSavedResults = function(jobId) {
+        deleteSavedResults(jobId);
+        vm.savedResults = loadSavedResults();
+    }
+
+    vm.deleteAllSavedResults = function() {
+        vm.savedResults = []
+        updateSavedResults(vm.savedResults);
+    }
+
+    vm.reloadSavedResults = function(jobId) {
+        loading.load();
+
+        query.executeQuery(
+            {
+              "class": "uk.gov.gchq.gaffer.operation.impl.export.resultcache.GetGafferResultCacheExport",
+              "jobId": jobId
+            },
+            function(data) {
+                submitResults(data);
+            }
```

Comment:
May want to handle errors here (eg if the operation isn't supported)

Reply:
The error is handled fairly well by the query.executeQuery function. The error would be something like:
Operation class uk.gov.gchq.gaffer.operation.impl.export.resultcache.GetGafferResultCacheExport is not supported by the MockAccumuloStore.